levels/classic_cfgs/rules.cfg

```diff
@@ -3,3 +3,5 @@
 
 [game]
 DungeonHeartHealHealth = 1
+; Total Amount of Creatures on the map .The absolute maximum is 1024.
+MapCreatureLimit =  256
```

Comment:
I would much prefer to have the actual limit here. Just set it to 255, and when loading the config or script command add the +1.